eng/common/scripts/stress-testing/stress-test-deployment-lib.ps1

```diff
@@ -386,8 +387,14 @@ function generateRetryTestsHelmValues ($pkg, $releaseName, $generatedHelmValues)
     $podOutput = RunOrExitOnFailure kubectl get pods -n $pkg.namespace -o json
     $pods = $podOutput | ConvertFrom-Json
 
+    # helm version example: v3.11.2+g912ebc1
+    $helmVer = (helm version --short).substring(1) -replace '\+.*',''
+    $helmVer = [AzureEngSemanticVersion]::new($helmVer)
+    $minHelmVer = [AzureEngSemanticVersion]::new("3.11.0")
+    if ($helmVer.CompareTo($minHelmVer) -le 0) {
+        throw "Please update helm to version >= 3.11.0`nAdditional information for updating helm version can be found here: https://helm.sh/docs/intro/install/"
+    }
```

Comment:
The `AzureEngSemanticVersion` class implements `IComparable` so you can just do `-lt`. Also I suggest you rename the variables to be more readable, no good reason to shorten them here, as well as adding a `$MIN_HELM_VERSION` constant up at the top of the file so it's easier to update.

```suggestion
    # helm version example: v3.11.2+g912ebc1
    $helmVersionString = (helm version --short).substring(1) -replace '\+.*',''
    $helmVersion = [AzureEngSemanticVersion]::new($helmVersionString)
    $minHelmVersion = [AzureEngSemanticVersion]::new($MIN_HELM_VERSION)
    if ($helmVersion -lt $minHelmVersion) {
        throw "Please update helm to version >= $MIN_HELM_VERSION (current version: $helmVersionString)`nAdditional information for updating helm can be found here: https://helm.sh/docs/intro/install/"
    }
```